flow/genericactors.actor.h

```diff
@@ -881,6 +881,33 @@ Future<T> ioTimeoutError( Future<T> what, double time ) {
 	}
 }
 
+ACTOR template <class T>
+Future<T> ioTimeoutError( Future<T> what, double errTime, Reference<AsyncVar<bool>> degraded, double degradedTime ) {
```

Comment:
Is there any situation that a disk's degratedTimeOut value should be larger than disk's IO failure? 
IIUC, a disk degrades before it fails. 

The function name can be changed to `ioDegradedOrTimeoutError`(), which better reflect what it does.

Reply:
if you wanted to disable the degraded behavior you would set the knob for its timeout to a large value